Runtime/Plugins/Raycast/RaycastableContainer.cs

```diff
@@ -0,0 +1,118 @@
+using Unity.UIWidgets.foundation;
+using Unity.UIWidgets.rendering;
+using Unity.UIWidgets.ui;
+using Unity.UIWidgets.widgets;
+
+namespace Unity.UIWidgets.plugins.raycast {
+    class RaycastableBox : SingleChildRenderObjectWidget {
+        public RaycastableBox(
+            Key key = null,
+            Widget child = null
+        ) : base(key, child) {
+            this.windowHashCode = Window.instance.GetHashCode();
+        }
+
+        readonly int windowHashCode;
+
+        public override RenderObject createRenderObject(BuildContext context) {
+            return new RenderRaycastableBox(
```

Comment:
why not pass widget.child to the renderObject ?

Reply:
I think we don't need the child info to do some calculations in the `RenderRaycastableBox` `paint` function, and I found that the `DecoratedBox` class (which I refer to) also doesn't pass its child widget in the `createRenderObject` function